### DISCUSSION TITLE
Unable to use search to find PDF or similar files within a repo using the path qualifier

### DISCUSSION THREAD

**Discussion by ababiec** [community/community#116489] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Assume you have a repo that contains some pdf files within the repo's contents, buried in one or more folders.

If you try to use path:*.pdf in the github.com search, it will report "Your search did not match any code"

This also applies to other binary documentation formats: *.docx, *.pptx, etc.

Granted, pdf files are not code but I am not expecting the contents to be searched, just let me know if the file(s) exists and where they are located.
This used to work in the prior search implementation, and it is disappointing that this functionality is broken. I am unable to find anything in the new search documentation that explains why this is not working, if it was intentional.

here is an example of a repo with a pdf file in the root folder and search is unable to find the file...
https://github.com/search?q=repo%3APacktPublishing%2FStreaming-Big-Data-with-Spark-Streaming-Scala-and-Spark-3-%20path%3A*.pdf&type=code

**Comment by look** [comment, marked as answer]
[Binary files are excluded](https://docs.github.com/en/search-github/github-code-search/about-github-code-search#limitations) from code search.

We would like to index their paths so searches like this will work, but I can't say when we'll be able to get to that.

**Reply by ababiec** [reply]
Thank you for the response. Disappointing but at least I know now why this is not working.

I have opened a PR to improve the documentation to clearly state this limitation in the Limitations section. I have read thru all the code search documents and I was unable to find any reference to binary files being excluded.

https://github.com/github/docs/pull/32379

**Reply by look** [reply]
Thanks. It's covered by "Only UTF-8 encoded files are included" since binary files are not UTF-8, but I agree that it's worth calling out separately for clarity.
